tests/e2e/client/apiproxy_bookstore_quota_test.py

```diff
@@ -45,29 +46,34 @@ def verify_quota_control(self):
     FLAGS.verbose = False
     self.set_verbose(FLAGS.verbose)
 
+
+    def _try_call_quota_read():
+      try:
+        return self._call_http(path='/quota_read',
+                                   api_key=FLAGS.api_key)
+      except ssl.SSLError as e:
```

Comment:
Why do only the quota requests get this exception? Other requests don't?

Reply:
I remember I have seen this before and other tests using same util lib doesn't call same api so frequently.